tests/fixtures/integration/actions/run/ping.yml

```diff
@@ -0,0 +1,5 @@
+- hosts: localhost
+  gather_facts: false
+  tasks:
+    - name: ensure we can run a ping task
+      ping:
```

Comment:
Can we add `vars_prompt` scenario. It is useful especially in EE case when the playbook is executed within a container but any stdin prompt from ansible-core has to be passed on to the local terminal for user input.

Reply:
Yeah, I'm planning to add more tests over the next few days, but wanted to keep this PR self contained. I'll open new PRs with more tests around `run` and also some of the other existing functionality. :)